docs/examples/index.rst

```diff
@@ -4,11 +4,19 @@ Examples
 We have several examples to illustrate various features of the code. These
 notebooks have been rendered here for the web, but the originals are
 found in the ``examples/`` directory of the package, and you can run them
-yourself!
+yourself! 
 
 .. toctree::
     :maxdepth: 1
     :glob:
 
     nb/*
 
+You can also try them out online directly from your browser: 
+
+.. image:: https://mybinder.org/badge_logo.svg
+ :target: https://mybinder.org/v2/gh/dwhswenson/contact_map/master?filepath=%2Fexamples
+
+(Note: the performance of the online servers can vary widely.)
```

Comment:
Go ahead and move this above the toctree -- as the contents grow, I think it'll be better to have this above the fold. Also, probably better to use an inline image, which can be accomplished in RST [using a substitution definition](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#substitution-definitions).

Reply:
done